<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactListWidget</class>
 <widget class="QWidget" name="ContactListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="contactStackedWidget">
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="2">
        <widget class="QWidget" name="toolWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="2" column="2">
           <widget class="QPushButton" name="addBtn">
            <property name="styleSheet">
             <string notr="true">border: 0px;
background: #ddd;
border-radius: 5px;
padding: 2px;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/images/addFriend.svg</normaloff>:/images/addFriend.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QWidget" name="searchWidget" native="true">
            <property name="styleSheet">
             <string notr="true">background: #ddd;
border-radius: 5px;</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="searchBtn">
               <property name="styleSheet">
                <string notr="true">border: 0px;
background: #ddd;
border-radius: 5px;
padding: 3px;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res.qrc">
                 <normaloff>:/images/search2.svg</normaloff>:/images/search2.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="searchEdit">
               <property name="styleSheet">
                <string notr="true">border: 0px;
background: #ddd;
border-radius: 5px 5px 0 0;</string>
               </property>
               <property name="placeholderText">
                <string>搜索</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">border: none;
background: #e3e3e3;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="nullWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="nullWidget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="toolWidget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
margin: 1px;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="addWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: #ddd;
border-radius: 5px;</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_2">
               <property name="styleSheet">
                <string notr="true">border: 0px;
background: #ddd;
border-radius: 5px;
padding: 3px;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res.qrc">
                 <normaloff>:/images/contact.svg</normaloff>:/images/contact.svg</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit">
               <property name="placeholderText">
                <string>微信号/手机号</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="cancelBtn">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;
border-radius: 5px;
padding: 2px;
background-color: rgb(244, 244, 244);
color: rgb(0, 0, 0)
</string>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
           <pointsize>8</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
  background: #e3e3e3;</string>
         </property>
         <widget class="QStackedWidget" name="searchStackedWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>250</width>
            <height>65</height>
           </rect>
          </property>
          <widget class="QWidget" name="page_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(244, 244, 244);</string>
           </property>
           <widget class="QPushButton" name="addFriendBtn">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>65</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>65</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>41</width>
              <height>41</height>
             </rect>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid rgb(7, 193, 96);
border-radius: 8px;
background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../res.qrc">:/images/search.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>55</x>
              <y>20</y>
              <width>41</width>
              <height>21</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">
background-color: transparent;</string>
            </property>
            <property name="text">
             <string>搜索:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_confirm">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>20</y>
              <width>72</width>
              <height>21</height>
             </rect>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color: rgb(7, 190, 95);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>228</x>
              <y>22</y>
              <width>21</width>
              <height>21</height>
             </rect>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../res.qrc">:/images/arrow-right.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="Line" name="line_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>1</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
  background: #ccc;</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <zorder>label</zorder>
           <zorder>label_2</zorder>
           <zorder>label_confirm</zorder>
           <zorder>label_4</zorder>
           <zorder>addFriendBtn</zorder>
           <zorder>line_2</zorder>
          </widget>
          <widget class="QWidget" name="page_4">
           <widget class="QLabel" name="label_option">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>1</y>
              <width>250</width>
              <height>65</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(244, 244, 244);
color: rgb(158, 158, 158);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;无法找到该用户，请检查你填写&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;的帐号是否正确。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="Line" name="line">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>1</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
  background: #ccc;</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
